site.py

```diff
@@ -315,11 +320,13 @@ def check_container(cid):
 # Updated using the function above.
 @app.route('/containers/<uuid>/status', methods=['GET'])
 def container_status(uuid):
-    cid = container_id(uuid)
-    url = '/check_container/' + cid[:16]
+    url = launch_container(uuid)
```

Comment:
Right, but calling [`launch_container`](https://github.com/swanijam/HaaS/blob/status-and-logs-fix/site.py#L187-L232) will blindly attempt to start a container even if there are no files present.